### ISSUE TITLE
Thoughts on backfills based on trade history

### ISSUE THREAD

**Issue by jorisw** [carlos8f/zenbot#2746]
### System information
- **Have I written custom code (as opposed to using zenbot vanilla)**: Yes, lots
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: macOS 11.5 + Ubuntu 20
- **Zenbot branch**: Forked long ago from unstable
- **NodeJS version**: 14

### Describe the problem

Zenbot heavily relies on backfills, based on retrieving trade history from exchanges. Zenbot also heavily relies on [ccxt](https://github.com/ccxt/ccxt), the library that unifies various crypto exchanges into a single unified API.

The manual of ccxt [stresses](https://ccxt.readthedocs.io/en/latest/manual.html#public-trades) that in general you should not rely on exchanges being able or willing to supply trade history far into the past. Exchanges that don't offer this, generally do offer long candlestick histories, that basically aggregate the same data into 1m to 1M periods.

Long story short, the reliance of Zenbot on trades, rather than candles for historical data, is probably a critical design flaw.

I'm curious to know if current maintainers (if any) of Zenbot concur.
